extensions/blocks/videopress/editor.js

```diff
@@ -22,12 +51,24 @@ const addVideoPressSupport = ( settings, name ) => {
 	}
 
 	const { attributes, deprecated, edit, save, supports, transforms } = settings;
-
 	const { available, unavailableReason } = getJetpackExtensionAvailability( 'videopress' );
 
+	// Check if VideoPress is unavailable and filter the mediaplaceholder to limit options
+	if ( isSimpleSite && [ 'missing_plan', 'unknown' ].includes( unavailableReason ) ) {
```

Comment:
`isSimpleSite` is a method so needs to be invoked here with `isSimpleSite()`

Reply:
🤦‍♂ Thanks :)